test/test-fs-event.c

```diff
@@ -1121,7 +1121,7 @@ TEST_IMPL(fs_event_getpath) {
     ASSERT_EQ(r, UV_EINVAL);
     r = uv_fs_event_start(&fs_event, fail_cb, watch_dir[i], 0);
     ASSERT_OK(r);
-    len = 0;
+    len = 1;
     r = uv_fs_event_getpath(&fs_event, buf, &len);
```

Comment:
So, somehow, this is a breaking change, right? Probably it doesn't matter, but just pointing it out

Reply:
It's trading a bug for another I guess.

`r = uv_fs_event_getpath(&fs_event, NULL, &len);` would happily `memcpy` into `NULL` if there was enough space in `len`.

Now all functions will behave the same.